system-addon/lib/SectionsManager.jsm

```diff
@@ -162,6 +170,13 @@ const SectionsManager = {
               description: card.description,
               previewImageURL: card.image
             });
+            this.emit(this.DISPATCH_TO_MAIN, {
```

Comment:
I'm not sure what's the expected / allowed `emit`s. Wherever we have this behavior, it will still want a comment as previously requested.